eng/Versions.props

```diff
@@ -31,7 +31,7 @@
   <PropertyGroup Label="Dependencies from dotnet/core-setup">
     <MicrosoftNETCoreAppRefPackageVersion>3.0.0</MicrosoftNETCoreAppRefPackageVersion>
     <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.0.2</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
-    <MicrosoftNETCoreAppInternalPackageVersion>3.0.2-servicing-19604-02</MicrosoftNETCoreAppInternalPackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>3.0.2</MicrosoftNETCoreAppInternalPackageVersion>
```

Comment:
This shouldn't be changed.  I'm investigating why it needed to change.

Reply:
Uh oh, I may need to revert this change on every other PR I made. We'll need to figure out why this is 404ing the build first.